docker-compose.yml

```diff
@@ -656,13 +656,15 @@ services:
       # ksqlDB
       CONTROL_CENTER_KSQL_WIKIPEDIA_URL: http://ksqldb-server:8088
       # Access ksqlDB server from a browser on your localhost
-      CONTROL_CENTER_KSQL_WIKIPEDIA_ADVERTISED_URL: http://localhost:8088
+      # To use Control Center and its ksqlDB interface over https, before launching, run in the launch shell:
```

Comment:
Will this option be documented anywhere else besides in a comment here?

Reply:
@mikebin Agree, IMO the whole flow to C3 HTTPS warrants a separate section in docs.  @javabrett will you do this? Or  I'm happy to assist with the docs part if you prefer.

Reply:
I'm happy to contribute to docs - either in a separate PR or this one, but I won't be able to progress docs much this week.

Reply:
@javabrett unless you're pressed to get the functionality, let's keep this PR open till we get docs added in as well.  I'm happy to assist with the docs piece if it helps.